labs/consultant/02-Installation.md

```diff
@@ -84,7 +84,8 @@ This will allow you to log in with the credentials admin/admin
 
 Pelorus gets installed via helm charts. The first deploys the operators on which Pelorus depends, the second deploys the core Pelorus stack and the third deploys the exporters that gather the data. The below instructions install into a namespace called `pelorus`.
 
-    oc create namespace pelorus
+    cd pelorus
+    oc new-project pelorus
```

Comment:
I'm happy to have `cd pelorus` in there, but i'd prefer to keep the k8s native way of creating namespaces, not projects

Reply:
@jtudelag

Reply:
mmm..Shouldn't we use kubectl there instead? I mean, we are using oc and not taking full advantage of it...

Anyway, just nitpicking ;).